android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.novoda.android">
+
+    <application android:allowBackup="true"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
```

Comment:
If they have to set it as false then make sure to update the usage
documentation

On Tuesday, 1 March 2016, Paul Blundell <notifications@github.com> wrote:

> In android/src/main/AndroidManifest.xml
> <https://github.com/novoda/notils/pull/68#discussion_r54548691>:
>
> > @@ -0,0 +1,11 @@
> > +<manifest xmlns:android="http://schemas.android.com/apk/res/android"
> > +    package="com.novoda.android">
> > +
> > +    <application android:allowBackup="true"
> > +        android:label="@string/app_name"
> > +        android:supportsRtl="true"
>
> good question. no any downstream manifest has priority so they can set it
> to false
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/notils/pull/68/files#r54548691>.
>

-- 
Disclaimer: The information in this e-mail and any attachments is the 
property of Novoda Ltd and is confidential and may be legally privileged. 
This e-mail is intended solely for the person or organisation to which it 
is addressed. Any disclosure, copying or other use of the information by 
any person or organisation who is not the intended recipient is strictly 
prohibited and may be unlawful. If you have received this e-mail in error, 
please inform the sender immediately and delete/destroy this e-mail and any 
copies of it. Novoda Ltd has taken reasonable precautions to minimise the 
risk of any software viruses which may damage your systems, but we advise 
that you take the necessary steps to ensure that no virus contamination is 
suffered. Novoda Ltd does not accept any liability for any loss or damage 
caused by the transmission of any virus.

Novoda Ltd, Company No: 347444, Registered in Scotland Registered 
Office: C/O Alexander Sloan, 38 Cadogan Street, Glasgow, G2 7HF, Scotland.  VAT 
Registration Number  GB 984 2525 93